docs/how-to/setup-time-sources.md

```diff
@@ -0,0 +1,28 @@
+# How to: Setup Time Sources
+
+Chrony alone cannot provide time; it requires another time source, either a 
+reference clock or another NTP server, to supply time to Chrony itself. 
+Currently, the Chrony charm supports only using another NTP server as the 
+time source.
+
+## Use Another NTP Server
```

Comment:
The [style guide](https://docs.ubuntu.com/styleguide/en/#headings) recommends using sentence case for headings, in other words:
`## Use another NTP server`

Reply:
Updated alongside other headings, thanks!